### PR TITLE
[11.0][MIG] mrp_sale_info: Migration to 11.0

### PR DESCRIPTION
**Pull request by oihane**

### CODE REVIEW

**Comment by oihane** [comment]
@OCA/manufacturing-maintainers can anyone check this please, thanks in advance

**Review by bodedra — APPROVED** [review]
Functional review. :+1:

File: mrp_sale_info/__manifest__.py [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "MRP Sale Info",
     "summary": "Adds sale information to Manufacturing models",
-    "version": "9.0.1.0.0",
+    "version": "11.0.1.0.0",
     "category": "Manufacturing",
-    "website": "http://www.antiun.com",
+    "website": "https://github.com/oca/manufacture",
     "author": "Antiun Ingeniería S.L., "
```
Comment (pedrobaeza) [inline mrp_sale_info/__manifest__.py]: Please remove Antiun, OdooMRP team, Serv. Tecnol... and add Tecnativa

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: mrp_sale_info/__manifest__.py (hunk 1 above)
Comment (oihane) [reply mrp_sale_info/__manifest__.py]: Change done

**Review by pedrobaeza — APPROVED** [review]

**Comment by gustavovalverde** [comment]
This used to have a relation to sale order lines. Was it removed on purpose or based on an actual limitation?

**Comment by anajuaristi** [comment]
I think the correct behavior should be having a link with sale.order.line and so ... related to sale_id 
I don't think it's complex to change on the module itselft but... hook needed to fix old data updating the module.
@oihane @pedrobaeza what do you think? should we afford this on V11 or change directly when migrating to V12?

**Comment by pedrobaeza** [comment]
Yes, I agree

**Pull request merged**